lib/build.js

```diff
@@ -570,11 +570,15 @@ const buildAct = config => ({act, oog, pass, name, hasGas, gas, lemma}) => {
   const buildAccount = varname => {
 
     const isPass = pass && !oog;
+    const origStorage_str = Object.keys(act.storage[varname])
```

Comment:
```suggestion
   // This means that we're assuming the LHS of the term has the same value as it did at the beginning of the transaction.
   // This assumption doesn't hold when the storage location has been modified in another call within a transaction.
   // When that happens, claims about gas for this storage locatoin won't be completely accurate.
   // This could be solved by this value symbolic, which it how it used to be.
    const origStorage_str = Object.keys(act.storage[varname])
```